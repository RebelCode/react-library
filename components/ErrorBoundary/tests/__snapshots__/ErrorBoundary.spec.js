import React from "react";
import { render, cleanup } from "@testing-library/react";
import { ErrorBoundary } from "../../index";

afterEach(cleanup);

describe("ErrorBoundary", () => {
  it("should take a snapshot", () => {
    const { asFragment } = render(
      <ErrorBoundary />
    );

    expect(asFragment(<ErrorBoundary />)).toMatchSnapshot();
  });
});
