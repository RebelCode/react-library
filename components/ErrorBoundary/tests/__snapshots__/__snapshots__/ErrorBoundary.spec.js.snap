// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorBoundary should take a snapshot 1`] = `<DocumentFragment />`;
